types/knex-cleaner/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

Comment:
I think this one is not necessary, especially if `console.log` is dropped from the tests.

Reply:
Yeah, I added this solely due to `console.log` in tests

Reply:
Removed, tests no longer use `console.log` so there is no longer any need for this